src/FirestoreApp.js

```diff
@@ -51,5 +56,16 @@ export function FirestoreApp(props) {
     }
 
     const taskData = loading ? loading : value.docs.map((doc) => doc.data());
```

Comment:
need to check value is not undefined.  Could be that loading is false, value is undefined and error is some error.